config.js

```diff
@@ -6,28 +6,17 @@ var LIB_DEST_PATH = config.LIB_DEST_PATH;
 
 var localConfig = extend(true, {
     bowerConfig: {
-        // Bower configuration for which files to get, and where to put them.
-        // [Source, excluding bower_components]: [Destination].
-        // 'isotope/dist/isotope.pkgd.js': config.LIB_DEST_PATH,
+        'document-register-element/build/document-register-element.max.js': config.LIB_DEST_PATH,
```

Comment:
I'd like the copy step to go away. The webserver serves bower_components at `/` so this file should also be accessible at `/document-register-element/build/document-register-element.max.js`. That sucks so you can set `rewriteMiddleware` instead.

I don't think `shim` is required for this, but it can't hurt I guess.

```js
var localConfig = extend(true, {
    rewriteMiddleware: [
        {from: '/media/js/lib/document-register-element.js',
         to: '/document-register-element/build/document-register-element.max.js'},
    ],
    requireConfig: {
        paths: {
            'document-register-element': 'lib/document-register-element',
        },
    },
    // ...
}, config);